client/SMCE_Client.cpp

```diff
@@ -0,0 +1,339 @@
+        return EXIT_FAILURE;                                                                                                     
+    }                                                                                                                                                                                                                   
+    board.attach_sketch(sketch);                                                                                                 
+    // Create Board Config                                                                                                       
+    smce::BoardConfig board_conf{                                                                                                
+        .pins = {0,1}, //Creating pins & GPIO drivers                                                                            
+        .gpio_drivers = {                                                                                                        
+            smce::BoardConfig::GpioDrivers{0,                                                                                    
+            smce::BoardConfig::GpioDrivers::DigitalDriver{true, true},                                                           
+            smce::BoardConfig::GpioDrivers::AnalogDriver{true, true}                                                             
+            },                                                                                                                   
+            smce::BoardConfig::GpioDrivers{1,                                                                                    
+            smce::BoardConfig::GpioDrivers::DigitalDriver{false, true},                                                          
+            smce::BoardConfig::GpioDrivers::AnalogDriver{false, true}                                                            
+            }                                                                                                                    
+        },                                                                                                                       
+        .uart_channels = { {} }, // use standard configuration of uart_channels                                                  
+        .sd_cards = { smce::BoardConfig::SecureDigitalStorage{ .root_dir = arduino_root_dir } }                                               
+    };                                                                                                                                                                                                            
+    board.configure(std::move(board_conf));                                                                                      
+    return 0;                                                                                                                    
```

Comment:
At line 96 you return `EXIT_FAILURE`, so logically here you should return `EXIT_SUCCESS` for coherence